source/_docs/pantheon-workflow.md

```diff
@@ -12,13 +12,17 @@ Every Pantheon site comes with three environments: Dev, Test, and Live. Separate
   <p>Optimize your dev team and streamline internal workflows. Pantheon delivers custom workshops to help development teams master our platform and improve their internal DevOps.</p>
 </div>
 
-## Code Moves Up, Content Moves Down
-<img src="/source/docs/assets/images/workflow.png" alt="Dev Test and Live icon" style="border:0;margin-left:auto;margin-right:auto;display:block;">
+## Code VS Content
+
+One of the core concepts at the heart of the Pantheon workflow is the distinction between **code** and **files**.
 
-The core of the Pantheon workflow is to move code up from Dev to Test to Live and content down from Live to Test to Dev.
+ - **Code** is the core of your CMS, plugins/modules, themes, css, javascript - anything that builds and shapes your site.
```

Comment:
Suggest reverting "anything that builds and shapes your site" to something closer to the original, which specifically noted that code is tracked by Git.

Reply:
Can we show them something here? A code directory structure in a drop down panel maybe? To help connect the dots visually?

Reply:
Maybe these should be tabs? instead of bullets?